docs/apis/http/webdav/_index.md

```diff
@@ -0,0 +1,533 @@
+
+```xml
+<?xml version="1.0"?>
+    <d:propfind xmlns:d="DAV:" xmlns:oc="http://owncloud.org/ns">
+    </d:propfind>
+```
+
+### Supported WebDAV Properties
+
+| Property                    | Desription                                                                 | Example                                                                                                                                          |
+|-----------------------------|----------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------|
+| `<d:creationdate />`        | The creation date of the resource.                                         | `1970-01-01T00:00:00+00:00`                                                                                                                      |
+| `<d:getlastmodified />`     | The latest modification time.                                              | `Fri, 30 Dec 2022 14:22:43 GMT`                                                                                                                  |
+| `<d:getetag />`             | The file's etag.                                                           | `"c3a1ee4a0c28edc15b9635c3bf798013"`                                                                                                             |
+| `<d:getcontenttype />`      | The mime type of the file.                                                 | `image/jpeg`                                                                                                                                     |
+| `<d:resourcetype />`        | Specifies the nature of the resource.                                      | `<d:collection />` for a folder                                                                                                                  |
+| `<d:getcontentlength />`    | The size if it is a file in bytes.                                         | `5` bytes                                                                                                                                        |
+| `<d:lockdiscovery />`       | Describes the active locks on a resource.                                  | Detailed Example in [Locking]()                                                                                                                  |
+| `<oc:id />`                 | The globally unique ID of the resource.                                    | `storage-1$27475553-7fb7-4689-b4cf-bbb635daff79!27475553-7fb7-4689-b4cf-bbb635daff79`                                                            |
+| `<oc:fileid />`             | The globally unique ID of the resource.                                    | `storage-1$27475553-7fb7-4689-b4cf-bbb635daff79!27475553-7fb7-4689-b4cf-bbb635daff79`                                                            |
```

Comment:
what different between **fileid** and **id**. I see both holding the same value. Could you help me here

Reply:
there is no difference